tests/__init__.py

```diff
@@ -3,3 +3,6 @@
 
 PATH = os.path.dirname(os.path.abspath(__file__))
 sys.path.insert(0, PATH + '/../serverlessrepo/')
+
+# set expected aws region environment variable
+os.environ['AWS_DEFAULT_REGION'] = 'us-east-1'
```

Comment:
I recommend putting this in `conftest.py` instead of `__init__.py`. `conftest.py` is a special file that py.test loads and can be used for common fixtures/module imports. More info here:

https://stackoverflow.com/questions/34466027/in-py-test-what-is-the-use-of-conftest-py-files

The reason I prefer `conftest.py` to putting this in `pytest.ini` is because then I can have different env vars/etc for unit and integ tests. I've had trouble getting py.test to use different `pytest.ini` files so having `test/unit/conftest.py` and `test/integration/conftest.py` has worked well for me as an alternative.

Reply:
Oh wait. I forgot we're using unittest instead of py.test for this module? If that's correct then you can disregard my comment.

Reply:
Moved the config to conftest.py.